packages/dapp-svelte-wallet/react-ui/src/components/NavMenu.js

```diff
@@ -0,0 +1,80 @@
+import { useMemo, forwardRef } from 'react';
+import { Link, useRouteMatch } from 'react-router-dom';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import { ListItemText, ListItemIcon } from '@material-ui/core';
+import DashboardIcon from '@material-ui/icons/Dashboard';
+import AccountBalanceWallet from '@material-ui/icons/AccountBalanceWallet';
+import Apps from '@material-ui/icons/Apps';
+import People from '@material-ui/icons/People';
+import AddCircle from '@material-ui/icons/AddCircle';
+
+const useStyles = makeStyles(theme => ({
+  nav: {
+    position: 'fixed',
+    top: theme.appBarHeight,
+    width: theme.navMenuWidth,
+    height: `calc(100vh - ${theme.appBarHeight})`,
+    overflowY: 'auto',
+  },
+  sectionHeader: {
+    padding: '16px',
+    fontFamily: ['Montserrat', 'Arial', 'sans-serif'].join(','),
```

Comment:
Omitting the double-quotes from `'"Montserrat"'`, etc seems inconsistent with the other `fontFamily` style.

Reply:
Fixed in the other place